csharp/src/IceLocatorDiscovery/PluginI.cs

```diff
@@ -702,8 +702,8 @@ public void
                 properties.setProperty(_name + ".Locator.AdapterId", Guid.NewGuid().ToString());
             }
 
-            _replyAdapter = _communicator.createObjectAdapter(_name + ".Reply");
-            _locatorAdapter = _communicator.createObjectAdapter(_name + ".Locator");
+            _replyAdapter = _communicator.createObjectAdapter(_name + ".Reply", authenticationOptions: null);
```

Comment:
Ditto.

If we need to revisit this code, add a TODO.

Reply:
- Locator adapter: this adapter hosts the locator implemented by the plug-in. My understanding is that we don't set endpoints by default and that we only use it for collocated calls. This adapter is not documented in the [3.7 version](https://doc.zeroc.com/ice/latest/property-reference/icediscovery), but it is in the [3.6 version](https://doc.zeroc.com/ice/3.6/property-reference/icediscovery#id-.IceDiscovery.*v3.6-IceDiscovery.Locator.AdapterProperty)
- Reply adapter: default to UDP endpoints, It would be rare to manually set the endpoint but I guess is possible.
- IceDiscovery.Multicast.Discovery: This is for receiving multicast requests, this adapter would not use SSL.

 I think is fine to not provide custom server authentication options for these adapters.